snap/hooks/install-device

```diff
@@ -9,3 +10,8 @@ net.ipv4.neigh.default.gc_thresh2=15360
 net.ipv4.neigh.default.gc_thresh3=16384
 net.netfilter.nf_conntrack_max=1048576
 EOF
+
+
+cat << EOF > "/run/mnt/ubuntu-data/system-data/_writable_defaults/etc/modules-load.d/aws-gadget.conf"
+nf_conntrack
+EOF
```

Comment:
You could use https://snapcraft.io/docs/kernel-module-load-interface instead

Reply:
thx! done